toolchain.py

```diff
@@ -226,35 +226,35 @@ def get_env(self):
 
 
 
-class ArchSimulator(Arch):
-    sdk = "iphonesimulator"
-    arch = "i386"
-    triple = "i386-apple-darwin11"
-    version_min = "-miphoneos-version-min=6.0.0"
-    sysroot = sh.xcrun("--sdk", "iphonesimulator", "--show-sdk-path").strip()
+#class ArchSimulator(Arch):
```

Comment:
Don't comment, just remove the section. If we want to be "safe", we can create a branch before this commit for back compatibility.

Reply:
right. Removed in latest commit.